tests/test_overscanCorrection.py

```diff
@@ -106,7 +111,6 @@ def checkPolyOverscanCorrectionX(self, **kwargs):
 
         dataBox = lsst.geom.Box2I(lsst.geom.Point2I(0, 0), lsst.geom.Extent2I(10, 10))
         dataImage = afwImage.MaskedImageF(maskedImage, dataBox)
-
```

Comment:
I don't remember why that's there, probably I was testing some code, or put a "print" statement and then deleted it. I don't care either; why is it important?

Reply:
I think it's just generally untidy and making diffing more confusing. @timj are there reasons to care deeply about such things?

Reply:
Usually removing or adding lines in parts of the code not being changed is not seen as worthwhile unless it's making the code compliant with some standard. I tend to avoid these sorts of changes because they distract people reading the diff into wondering why the line disappeared. If removing/adding the blank line is wanted it should be done as a distinct commit making clear this is a formatting cleanup.